src/Aspire.Dashboard/Components/Dialogs/FilterDialog.razor.cs

```diff
@@ -41,23 +43,35 @@ protected override void OnInitialized()
 
         _formModel = new LogDialogFormModel();
         EditContext = new EditContext(_formModel);
+    }
+
+    protected override void OnParametersSet()
+    {
+        var names = new List<string> { LogFilter.KnownMessageField, LogFilter.KnownCategoryField, LogFilter.KnownApplicationField, LogFilter.KnownTraceIdField, LogFilter.KnownSpanIdField, LogFilter.KnownOriginalFormatField };
+        var knownFields = names.Select(p => new SelectViewModel<string> { Id = p, Name = LogFilter.ResolveFieldName(p) }).ToList();
+        var customFields = Content.LogPropertyKeys.Select(p => new SelectViewModel<string> { Id = p, Name = LogFilter.ResolveFieldName(p) }).ToList();
+
+        _parameters =
+        [
+            .. knownFields,
+            new SelectViewModel<string> { Id = null, Name = "-" },
```

Comment:
Is there not a better way to create a separator with this control?

Reply:
Not that I know of - https://www.fluentui-blazor.net/Select